tests/integration/hpe_3par_api_manage_test.py

```diff
@@ -76,6 +76,7 @@ def setUpClass(cls):
                 assert pl_data['Enabled'] is False
                 while pl_data['Enabled'] is False:
                     c.enable_plugin(HPE3PAR)
+                    sleep(60)
```

Comment:
@shaik-gousa-khaja-asif introduce def to check whether all backends are up instead of using a sleep value. This can be applied at all places.